File: api/Proyecto_BK/Proyecto_BK.Common/Models/ComboPersonalViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Proyecto_BK.Common.Models
{
    public class ComboPersonalViewModel
    {
        public int Comb_Id { get; set; }
        public string Comb_Descripcion { get; set; }
        public decimal? Comb_Precio { get; set; }
        public string Comb_Imagen { get; set; }
        public int? Bebi_Id { get; set; }
        public int? Post_id { get; set; }
        public int? Comp_Id { get; set; }
        public int? Alim_Id { get; set; }
        public int? Comb_Usua_Creacion { get; set; }
        public DateTime? Comb_Fecha_Creacion { get; set; }
        public int? Comb_Usua_Modifica { get; set; }
        public DateTime? Comb_Fecha_Modifica { get; set; }
        public bool? Comb_Estado { get; set; }

        public string Bebi_Descripcion { get; set; }
        public string Post_Descripcion { get; set; }
        public string Comp_Descripcion { get; set; }
        public string Alim_Descripcion { get; set; }


        public string Usua_Usuario { get; set; }
        public int TotalPedidosCombos { get; set; }
        public int Paqe_Id { get; set; }
        public string Paqe_Descripcion { get; set; }
        public int TotalPedidosPaquetes { get; set; }
        public int TotalPedidosAlimentos { get; set; }
        public int TotalPedidosPostres { get; set; }
    }
}

File: app/plantilla/src/app/demo/models/ComboPersonalViewModel.ts
export class ComboPersonal {
    comb_Id?: number;
    Comb_Descripcion?: string;
    Comb_Precio?: string;
    Comb_Imagen?: string;
    Bebi_Id?: string;
    Post_Id?: string;
    Comp_Id?: string;
    Alim_Id?: string;
    Comb_Usua_Creacion?: string;
    Comb_Fecha_Creacion?: string;
    Comb_Usua_Modifica?: string;
    Comb_Fecha_Modifica?: string;

}

export class ComboPEnviar {
    Comb_Id?: number;
    Comb_Descripcion?: string;
    Comb_Precio?: string;
    Comb_Imagen?: string;
    Bebi_Id?: string;
    Post_Id?: string;
    Comp_Id?: string;
    Alim_Id?: string;
    Comb_Usua_Creacion?: number;
    Comb_Usua_Modifica?: number;
}

export class Llenar{
    comb_Id: number;
    alim_Id: number;
    bebi_Id: number;
    comp_Id: number;
    post_Id: number;
    usua_Creacion: string;
    usua_Modifica: string;
    comb_Usua_Creacion: number;
    comb_Fecha_Creacion: string;
    comb_Fecha_Modificacion: string;
    comb_Descripcion: string;
    comb_Precio: string;
    comb_Imagen: string;
    bebi_Descripcion: string;
    post_Descripcion: string;
    comp_Descripcion: string;
    alim_Descripcion: string;
}

export class Fill {
    comb_Id?: string;
    comb_Descripcion?: string;
    comb_Precio?: string;
    comb_Imagen?: string;
    bebi_Id?: string;
    post_Id?: string;
    comp_Id?: string;
    alim_Id?: string;

    bebi_Descripcion?: string;
    post_Descripcion?: string;
    comp_Descripcion?: string;
    alim_Descripcion?: string;

    usua_Creacion: string;
    usua_Modifica: string;
    comb_Usua_Creacion?: string;
    comb_Fecha_Creacion?: string;
    comb_Usua_Modifica?: string;
    comb_Fecha_Modifica?: string;
}


